guides/common/modules/con_resolving-package-dependencies.adoc

```diff
@@ -1,56 +1,67 @@
+For example, dependency solving is useful when you incrementally add a single package to a Content View version.
+You might need to enable dependency solving to install that package.
 
-The package dependency resolution feature does not take into account which packages are installed on your clients.
-Dependency solving considers only packages within the repositories of the Content View.
+However, dependency solving is unnecessary in most situations. For example:
+
+* When incrementally adding a security errata to a Content View, dependency solving can cause significant delays to Content View publication without major benefits.
+* In case of an outdated Content View version, the packages in a newer erratum might depend on packages outside that Content View.
+Incrementally adding the erratum using dependency solving might include unwanted packages.
+As an alternative, consider updating the Content View.
+
+[NOTE]
+====
+Dependency solving only considers packages within the repositories of the Content View.
+It does not consider packages installed on clients.
 ifeval::["{client-os-family}" == "Red Hat"]
-For example, if a Content View has only AppStream in it, dependency solving will not know to pull dependent BaseOS content in at publish time.
+For example, if a Content View includes only AppStream, dependency solving does not include dependent BaseOS content at publish time.
 endif::[]
-For more information, see {ContentManagementDocURL}Limitations_to_Repository_Dependency_Resolution_content-management[Limitations to Repository Dependency Resolution] in _{ContentManagementDocTitle}_.
 
-.Resolving Package Dependencies and Filters
-Filters, by default, do not resolve any dependencies of the packages listed in the filters.
-This might require some level of testing to determine what dependencies are required.
+For more information, see {ContentManagementDocURL}Limitations_to_Repository_Dependency_Resolution_content-management[Limitations to Repository Dependency Resolution] in _{ContentManagementDocTitle}_.
+====
 
-If you add a filter that excludes some packages that are required and the Content View has dependency resolution enabled, {Project} ignores the rules you create in your filter in favor of resolving the package dependency.
+Dependency solving can lead to the following problems:
 
-If you create a content filter for security purposes but choose to resolve dependencies, {Project} can add packages that you might consider insecure.
+Dependency solving can significantly delay Content View publication::
+{Project} examines every repository in a Content View for dependencies.
+Therefore, publish time increases with more repositories.
++
+To mitigate this problem: Use multiple Content Views with fewer repositories.
+Combine them using composite Content Views.
```

Comment:
```suggestion
To mitigate this problem, use multiple Content Views with fewer repositories and combine them into composite Content Views.
```

Reply:
On a scale of 1-10, how much does this bother you? :) I wrote my first version as you suggest here and elsewhere (For example, ... To mitigate, ...). Then I changed it to use colons. My reasoning was this:

* I wanted to make the descriptions easier to scan quickly: using a colon creates a label of sorts that makes it easier to scan for similar pieces of information.
* It also counts as two short sentences rather than one long sentence (=> slightly better readability score).

Reply:
The IBM style guide which we generally use as documentation writing guide in downstream satellite recommends us to not use incomplete sentences. https://docs.engineering.redhat.com/display/CCSENABLE/Incomplete+sentences

Reply:
Thanks for explaining!